.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    csslint:
```

Comment:
`HINT`

Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml) instead.

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

Reply:
I used linter but id doesn't work and it gives me this message
![stickler](https://user-images.githubusercontent.com/14271563/63156858-5179cf00-c01e-11e9-9b65-6e9e672458f0.PNG)